Removed the hard coding of the viscosity properties in SetNondimensionalization.

vdweide opened the pull request:
Removed the hard coding of the viscosity properties to standard air when the Reynolds number is used to determine the free-stream values in CEulerSolver::SetNondimensionalization. For this reason, both dimensional and non-dimensional viscosity and thermal conductivity constants are now present in CConfig.

salvovitale commented on SU2_CFD/src/solver_direct_mean.cpp:
```diff
@@ -3818,30 +3826,10 @@ void CEulerSolver::SetNondimensionalization(CGeometry *geometry, CConfig *config
-          config->SetMu_Temperature_RefND(Mu_Temperature_RefND);
-          config->SetMu_SND(Mu_SND);
-        }
-        if (config->GetSystemMeasurements() == US) {
```
Well I think the second option is the most user friendly. We can do that in the config post-processing: if US units are used we convert the Sutherland default values to US system.

vdweide replied:
OK, I'll do that then.

Edwin

vdweide commented:
The default values for the viscosity coefficients are now corrected in CConfig::SetPostprocessing. I also corrected the constant value of thermal conductivity, where I assumed that the US units are BTU/hr/(feet ∗ F), leading to a conversion factor of 0.577789317. If this is not correct, let me know. 

For the US folks, isn't it about time that you switch to SI units?
